api/example_docs/m_15_16.xml

```diff
@@ -0,0 +1,367 @@
+        climate change, manage competing demands on environmental resources,
+        and ensure the Nation's security. 
+      </content>
+    </para>
+    <para>
+      <content>
+        Federal government funding for research and development (R&amp;D) is
+        essential to address societal needs in areas in which the private
+        sector does not have sufficient economic incentive to make the required
+        investments. Key among these is basic research- the fundamental,
+        curiositydriven inquiry that is a hallmark of the American research
+        enterprise and a powerful driver of new technology. Simply supporting
+        research is not sufficient, however, Federal agencies should ensure
+        that the results of that research are made available to other
+        scientists, to the public, and to innovators who can translate them
+        into the businesses and products that will improve all of our lives. 
+      </content>
+    </para>
+    <para>
+      <content>
+        This memorandum outlines the Administration' s multi-agency science and
+        technology priorities for formulating FY 2017 Budget submissions to the
+        Office of Management and Budget (OMB).  The priorities covered in this
+        memo require investments in R&amp;D; science, technology, engineering,
+        and mathematics (STEM) education; STEM workforce development;
+        technology transfer; R&amp;D infrastructure; and scientific-collection
+        management.  The priorities in this memorandum build on priorities
+        reflected in this Administration's past budgets and priorities
+        directives. 
+      </content>
+    </para>
+    <para>
+      <content>
+        Agencies should label the sections of their budget submissions that
+        address priorities described below.  Agencies engaged in complementary
+        activities should consult with each other during the budget planning
+        process to  coordinate resources, maximize impact, and avoid
+        inappropriate duplication, and they should include summaries ofthese
+        discussions in their OMB budget submissions.  Agency proposals aligned
+        with multi-agency R&amp;D priorities and demonstrating interagency
+        coordination are more likely to be prioritized in FY 2017 Budget
+        deliberations. 
+      </content>
+    </para>
+  </sec>
+  <sec>
+    <heading>Multi-agency R&amp;priorities</heading>
+    <para>
+      <content>
+        In the FY 2017 Budget, agencies should balance priorities to ensure
+        that resources are adequately allocated for agency-specific,
+        mission-driven research, including fundamental research, while focusing
+        resources, where appropriate, on the following multi-agency research
+        activities that cannot be addressed effectively by a single agency. 
+      </content>
+    </para>
+    <list>
+        <listitem emblem="1" marker="•">
```

Comment:
These'll be numbered automatically, so we probably don't need to spell out their emblem here.